CMakeLists.txt

```diff
@@ -642,11 +642,12 @@ hpx_option(HPX_WITH_ITTNOTIFY BOOL
 hpx_option(HPX_WITH_HPXMP BOOL
   "Enable hpxMP OpenMP emulation." OFF CATEGORY "OpenMP")
 if(HPX_WITH_HPXMP)
-  if(NOT MSVC AND ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang"))
+  if(NOT MSVC AND (("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang") 
+	OR ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")))
     hpx_option(HPX_WITH_HPXMP_NO_UPDATE BOOL
       "Do not update code from remote hpxMP repository." OFF CATEGORY "OpenMP")
   else()
-    hpx_warn("hpxMP can be used only when compiling with clang")
+    hpx_warn("hpxMP can be used only when compiling with clang or gcc")
```

Comment:
@hkaiser I guess this is the error message if we are not using clang or gcc, the one you commented on is a message when we define `HPX_WITH_HPXMP_NO_UPDATE `?

Reply:
ok, I take it back. I misunderstood the code.